jest.config.js

```diff
@@ -149,6 +149,7 @@ const jestConfig = {
     collectCoverageFrom: [
         // Code directories
         'packages/*/{src,lib}/**/*.js',
+        '!packages/*/{src,lib}/**/index.js',
```

Comment:
I added this line because our coverage numbers were checking index.js files which IMO do not warrant testing.

Reply:
Some of our packages other than `venia-concept` have real code in `index.js` though.

Reply:
Ok, should I make this more specific or remove it all together?